srcs/runners_manager/vm_creation/gcloud/GcloudManager.py

```diff
@@ -73,6 +105,13 @@ def configure_instance(
         )
         disk_size_gb = runner.vm_type.config["disk_size_gb"]
         disk_type = f"projects/{self.project_id}/zones/{self.zone}/diskTypes/pd-ssd"
+        labels = {}
+        for tag in runner.vm_type.tags:
+            if "small" in tag or "medium" in tag or "large" in tag:
+                labels["flavor"] = tag
+                break
+        labels["image"] = runner.vm_type.config["machine_type"]
```

Comment:
The machine_type is the flavor, not the image.

Let's use the variable available, defined by the user to build the image name:
`{runner.vm_type.config["project"]}-{runner.vm_type.config["family"]}`

Reply:
Done!